hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -5982,6 +5982,22 @@
     </description>
   </property>
 
+  <property>
```

Comment:
I am prefer to hard code and not necessary offer the parameter to end user to tune.